easybuild/easyconfigs/c/Clang/Clang-3.6.1-GCC-4.9.2.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = "Clang"
+version = "3.6.1"
+
+homepage = "http://clang.llvm.org/"
+description = """C, C++, Objective-C compiler, based on LLVM.  Does not
+ include C++ standard library -- use libstdc++ from GCC."""
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+# Do not set optarch to True: it will cause the build to fail
+toolchainopts = {'optarch': False}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = [
+    "llvm-%(version)s.src.tar.xz",
+    "cfe-%(version)s.src.tar.xz",
+    "compiler-rt-%(version)s.src.tar.xz",
+    "polly-%(version)s.src.tar.xz",
+]
+
+dependencies = [
+    ('GMP', '6.0.0'),
```

Comment:
hmm, doesn't that indicate that the GMP easyconfig file is misnamed, if the version is actually `6.0.0a` rather than `6.0.0`?